packages/snjs/lib/application.ts

```diff
@@ -145,6 +146,7 @@ export class SNApplication {
   private keyRecoveryService!: SNKeyRecoveryService;
   private preferencesService!: SNPreferencesService;
   private permissionsService!: SNPermissionsService;
+  private mfaService!: SnMfaService;
```

Comment:
```suggestion
  private mfaService!: SNMfaService;
```

Would go for consistency for now, until we decide to rename all classes to proper pascal case if that's what we want to do.

Reply:
I normally go by the Google's style guide on this[1](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case), [2](https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined), because it's the most reasonable and regular take on this that I know of.

But I'm ok with whatever is satisfactory to everyone else as long as there are some rules behind it that can be followed. 

So the rule I can detect in SNJS is to always capitalize SN, otherwise follow regular camelCase. Is this correct?

Outside of SNJS it definitely isn't (e.g. there are such names as VerifyMFA, VerifyMFADTO, HTTPClient) -- going by these the name would be `SNMFAService`, but this is not very readable indeed.

PS on that topic, the Web API built-in `XMLHttpRequest` is a nice illustration of camelCase/PascalCase being confusing -- that's probably why the Google guides have it as an example. :D

Reply:
Correct, the rule is capital SN, then PascalCase for the rest. The reason for the prefix btw is that each of the services can be subclassed by clients, so it makes it easier to detect what is the root class and what is the subclass. So for example mobile will subclass SNComponentManager as ComponentManager. Whereas if base class was ComponentManager the clients might have to do funky naming to avoid conflicting. (Or maybe they could just do WebComponentManager I suppose..)

Reply:
Also it was meant to allow clients to quickly see whether something came from SNJS or from the client code. But ultimately perhaps not necessary. So I'm in favor of generally refactoring this in bulk, but until we do, to have it blend in with the present style.

Reply:
Uppercased the N, so resolving.